samples/java/junit5-selenium/README.md

```diff
@@ -0,0 +1,28 @@
+# JUnit5 sample project
+
+## Pre-requisites for running the tests in the project
+
+- Java must be installed on your system (JDK8 or higher)
+- Maven must be installed on your system.
+- Python3 must be installed on your system.
+- Chrome browser must be installed on the system for running the tests . Make sure version of chromedriver placed in 'drivers' folder should match the version of locally installed Chrome. 
```

Comment:
maybe here add a reference to where to get the chromedriver.
also tried this out with macos, and it seems that it is looking for the chromedriver under the linux folder

Reply:
@bitcoder as discussed , I added a new folder for mac with appropriate drivers